### ISSUE TITLE
e is not defined

### ISSUE THREAD

**Issue by slayer** [gjritter/nerve#2]
```
ReferenceError: e is not defined
    at nerve/lib/nerve.js:95:7
    at node.js:303:21
    at node.js:762:9
```

**Comment by gjritter** [comment]
Do you have a test or example that you provide for this issue? The current set of tests are passing, so we probably need a new test that illustrates the problem.

**Comment by gjritter** [comment]
Added a test, and fixed in commit c94791a.

**Comment by gjritter** [comment]
Closed.
